pkg/kubectl/cmd/resource/get_test.go

```diff
@@ -50,6 +50,8 @@ import (
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi"
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
move this to line 29  between  "os" import and  "k8s" import

Reply:
Modified.